include/multipass/platform.h

```diff
@@ -57,13 +57,15 @@ class Platform : public Singleton<Platform>
     virtual bool is_remote_supported(const std::string& remote) const;
     virtual bool is_backend_supported(const QString& backend) const; // temporary (?)
     virtual int chown(const char* path, unsigned int uid, unsigned int gid) const;
+    virtual int chmod(const char* path, unsigned int mode) const;
     virtual bool link(const char* target, const char* link) const;
     virtual bool symlink(const char* target, const char* link, bool is_dir) const;
     virtual int utime(const char* path, int atime, int mtime) const;
     virtual QDir get_alias_scripts_folder() const;
     virtual void create_alias_script(const std::string& alias, const AliasDefinition& def) const;
     virtual void remove_alias_script(const std::string& alias) const;
     virtual std::string alias_path_message() const;
+    virtual void set_server_permissions(const std::string& server_address, const bool restricted) const;
```

Comment:
I think I'd rename this, it suggests that you can pass _any_ permissions. Maybe `set_server_restriction`? `set_server_protection`?

Reply:
Ok, sure.

Reply:
Ok, renamed.  Please let me know if this is good.